src/libsync/propagateupload.cpp

```diff
@@ -894,5 +898,392 @@ void PropagateUploadFile::abortWithError(SyncFileItem::Status status, const QStr
     done(status, error);
 }
 
+void PropagateBundleQNAM::start()
+{
+    QHttpMultiPart *multiPart = new QHttpMultiPart(QHttpMultiPart::RelatedType);
+    multiPart->setBoundary("boundrary123");
```

Comment:
Could be that it's not really necessary because it's already set: https://code.woboq.org/qt5/qtbase/src/network/access/qhttpmultipart.cpp.html#443
Maybe qDebug() the boundary to verify..